src/app/modules/home/__tests__/home.controller.spec.ts

```diff
@@ -0,0 +1,20 @@
+import * as HomeController from 'src/app/modules/home/home.controller'
+
+import expressHandler from 'tests/unit/backend/helpers/jest-express'
+
+describe('home.controller', () => {
+  describe('home', () => {
+    const mockReq = expressHandler.mockRequest()
+    const mockRes = expressHandler.mockResponse()
+    const mockNext = jest.fn()
+    HomeController.home(mockReq, mockRes, mockNext)
+
+    it('should call res.render with the correct arguments once', () => {
+      expect(mockRes.render).toBeCalledWith('index')
+      expect(mockRes.render).toHaveBeenCalledTimes(1)
+    })
+    it('should not call any downstream middleware', () => {
+      expect(mockNext).not.toHaveBeenCalled()
```

Comment:
I think the function call to `home` should be made inside the test function, and the two `it` blocks can be merged into one. Then future tests can vary the inputs to `home` without having to do additional refactoring.